### DISCUSSION TITLE
Lesson 6: Chain Id visible on anvil (version 0.1.0)

### DISCUSSION THREAD

**Discussion by itznishant** [Cyfrin/foundry-full-course-cu#418] [General]
[Lesson 6 Part 10]
So, in version 0.1.0 , anvil now shows the Chain Id as well, in (ref. image):

<img width="388" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/17523443/06f6c947-66f8-40c5-9c83-b4b742e9ec69">

@PatrickAlphaC 
Probably need to update the video "Lesson 6 Part 10 | Deploying to a local blockchain - Solidity & Foundry Full Course"  at below time.

https://youtu.be/IK2irq6_2fw?list=PL2-Nvp2Kn0FPH2xU3IbKrrkae-VVXs1vk&t=335

Thanks!

**Comment by yeahChibyke** [comment]
Why does Metamask keep throwing an error when I use that Chain ID. I even got this message when trying to set a currency symbol basically telling me to change it to GO.
![local_blockchain_error_1](https://github.com/Cyfrin/foundry-full-course-f23/assets/56419077/a0a94822-1781-49fa-9208-4c1a46e213ad)

**Reply by highHumann** [reply]
make sure your anvil is running in the background

**Comment by itznishant** [comment]
> 



> Why does Metamask keep throwing an error when I use that Chain ID. I even got this message when trying to set a currency symbol basically telling me to change it to GO. ![local_blockchain_error_1](https://user-images.githubusercontent.com/56419077/255463899-a0a94822-1781-49fa-9208-4c1a46e213ad.PNG)

HI, I see you have created a discussion for this already here.

`https://github.com/Cyfrin/foundry-full-course-f23/discussions/425`

For Lesson 6, I don't think you require to add chain to Metamask for anvil.

I can't help much here, but just running `anvil` locally in Win/ Mac should work fine.

**Reply by yeahChibyke** [reply]
I am unable to install Ubuntu locally (I am using a Windows-based laptop), hence I resorted to Gitpod
